addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17164,23 +17164,20 @@ msgstr ""
 
 #: xbmc/Application.cpp
 msgctxt "#34220"
-msgid "HDR OFF"
+msgid "HDR status"
 msgstr ""
 
 #: xbmc/Application.cpp
```

Comment:
Above this line, please add a comment for the translators explaining where and for what this string is used.